lib/cc/engine/csslint.rb

```diff
@@ -21,7 +21,12 @@ def run
             path = file['name'].sub(/\A#{@directory}\//, '')
             file.children.each do |node|
               next unless node.name == "error"
-              issue = create_issue(node, path)
+              issue =
+                if node.attributes.key?("identifier")
+                  create_issue(node, path)
+                else
+                  create_error(node, path)
```

Comment:
This approach introduces an unacceptable amount of syntactic duplication IMO. What do you think of instead doing the following:

- `check_name` is (basically) `node.identifier || "parse-error"`
- Have a `check_details` value for `parse-error`
- Everything else about `#create_issue` stays as-is
- No need for `#create_error`